builder/build.sh

```diff
@@ -4,7 +4,7 @@ source /build_environment.sh
 
 # Compile statically linked version of package
 echo "Building $pkgName"
-CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags '-s' $pkgName
```

Comment:
this is now set for the life of the container, if we are worried about this we could set a different ENV var and pass it in here, but this is a short lived container, so...